### PR TITLE
Concept Entry Javascript Accessors

### PR DESCRIPTION
**Pull request by Judis07**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/accessors/accessors.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+Title: 'Accessors'
+Description: "Provides access to an object's properties by using the dot notation or the bracket notation."
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'OOP'
+  - 'Methods'
```
Comment (droffilc1) [inline content/javascript/concepts/accessors/accessors.md:9]: To enhance the clarity of your code, consider adding more tags. For instance, including tags like 'Objects' and 'Classes' would help make the purpose and structure of your code even more apparent

File: content/javascript/concepts/accessors/accessors.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In JavaScript, object properties can be accessed using accessor known as getters. The `get` keyword is used to define a getter method for establishing custom behavior when accessing the values of object properties.
```
Comment (droffilc1) [inline content/javascript/concepts/accessors/accessors.md:15]: A JavaScript object property can be accessed using accessors known as getters. The `get` keyword is used to define a getter method for establishing custom behavior when accessing the values of object properties.

**Review by droffilc1 — CHANGES_REQUESTED** [review]
@Judis07 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by SSwiniarski** [comment]
@Judis07 after you're finished with the user review above, tag me and I'll do a first maintainer review.

File: content/javascript/concepts/accessors/accessors.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  - 'OOP'
+  - 'Methods'
+  - 'Objects'
+  - 'Classes'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In JavaScript, object properties can be accessed using accessor known as getters. The `get` keyword is used to define a getter method for establishing custom behavior when accessing the values of object properties, while the `set` keyword allows you to define a setter method to control the behavior when assigning values to those properties.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/accessors/accessors.md:17]:
```suggestion
In JavaScript, object properties can be obtained using **accessors**.  Properties are established and retrieved with methods known as "getters" and "setters". 

The `get` keyword is used to define a getter method for establishing a custom behavior when accessing the values of object properties, while the `set` keyword is used to create a setter method that determines the behavior when assigning values to those properties.
```

File: content/javascript/concepts/accessors/accessors.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    get propertyName() {}
+}
+
+object.propertyName
+object[propertyName]
+```
+
+## Example
+
+The following example defines a `User` class with a constructor and a getter method. It demonstrates how to encapsulate and access the GitHub profile information of a user using accessor methods.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/accessors/accessors.md:34]: I would suggest using a generic/fictional user and not your GitHub user profile

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Judis07 I've added several comments. Please review carefully, and update the entry. Tag me when you've finished making changes. Thanks

**Comment by Judis07** [comment]
@caupolicandiaz I have made all suggested changes that were mentioned. Please review again. Thanks

File: content/javascript/concepts/accessors/accessors.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+---
+Title: 'Accessors'
+Description: "Provides access to an object's properties by using the dot notation or the bracket notation."
```
[the remark this answers is not in the document]
Comment (Judis07) [reply content/javascript/concepts/accessors/accessors.md]: This was never my intention. I had no idea that my description matches with MDN, it was purely a coincident. However, i have corrected this and will be more careful about such things. Thanks for the heads up.

File: content/javascript/concepts/accessors/accessors.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Accessors'
+Description: "Offers access to an object's properties through either dot notation or bracket notation."
```
Comment (caupolicandiaz) [inline content/javascript/concepts/accessors/accessors.md:3]:
```suggestion
Description: 'Offers access to the properties of an object through either dot notation or bracket notation.'
```

Comment (caupolicandiaz) [reply content/javascript/concepts/accessors/accessors.md:3]: metadata must only use single quotes and there is no punctuation permitted within the description

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Judis07 A couple of additional edits. Please update. Thanks

**Comment by Judis07** [comment]
@caupolicandiaz changes are done please review.

**Comment by SSwiniarski** [comment]
@Judis07 can you update this with main so we can merge?

**Comment by Judis07** [comment]
@SSwiniarski update with main.

**Pull request merged**